src/hotspot/share/gc/g1/g1CodeRootSet.hpp

```diff
@@ -22,68 +22,59 @@
  *
  */
 
-#ifndef SHARE_GC_G1_G1CODECACHEREMSET_HPP
-#define SHARE_GC_G1_G1CODECACHEREMSET_HPP
+#ifndef SHARE_GC_G1_G1CODEROOTSET_HPP
+#define SHARE_GC_G1_G1CODEROOTSET_HPP
 
 #include "utilities/globalDefinitions.hpp"
 
+#include "code/codeCache.hpp"
+#include "memory/allocation.hpp"
+#include "utilities/resizeableResourceHash.hpp"
+#include "utilities/resourceHash.hpp"
+
+class CleanCallback;
```

Comment:
Do you need this forward declaration since it's in the .cpp file before the caller?